org.freedesktop.Platform.ffmpeg.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "org.freedesktop.Platform.ffmpeg",
+    "branch": "1.6",
+    "runtime": "org.freedesktop.Sdk",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "sdk-extensions": [],
+    "separate-locales": false,
+    "appstream-compose": false,
+    "cleanup": [
+        "/bin/ffmpeg",
+        "/lib/*.so",
+        "/include",
+        "/lib/pkgconfig",
+        "/share/ffmpeg/examples",
+        "/share/ffmpeg"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+        "cxxflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+        "ldflags": "-fstack-protector-strong -Wl,-z,relro,-z,now",
+        "prefix": "/usr/lib/ffmpeg",
+        "env": {
+            "V": "1"
+        }
+    },
+    "modules": [
+        {
+            "name": "ffmpeg",
+            "config-opts": [
+                /* Copied from runtiem */
+                "--disable-debug",
+                "--disable-doc",
+                "--disable-static",
+                "--enable-gpl",
+                "--enable-optimizations",
+                "--enable-libvpx",
+                "--enable-shared",
+                "--disable-ffplay",
+                "--disable-ffprobe",
+                "--disable-ffserver",
+                "--disable-everything",
+                "--enable-gnutls",
+                "--enable-libfontconfig",
+                "--enable-libfreetype",
+                "--enable-libopus",
+                "--enable-libpulse",
+                "--enable-libspeex",
+                "--enable-libtheora",
```

Comment:
I don't think it makes sense to include ENCODING support for obsolete codecs like theora or vorbis.

Reply:
I don't think --enable-libtheora means that, only that it can use the external library.
`ffmpeg -decoders` lists theora, but `ffmpeg -decoders` is empty.